style/app.scss

```diff
@@ -685,133 +640,133 @@ a:active
   background: #f0f5f8;
 }
 
-#sidebar
+.collapsed.sidebar
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  box-shadow : none;
+  border     : none;
 }
 
-#sidebar-button
+.sidebar
 {
-  cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #5c497e;
-  position         : absolute;
-  border           : solid 1px #000000;
+  display          : flex;
+  flex-direction   : column;
+  z-index          : 1;
+  position         : inherit;
+  width            : 160px;
+  background-color : #ffffff;
+  border-radius    : 10px;
+  box-shadow       : 4px 4px 4px #e8e9eb;
```

Comment:
I think the dimensions are good but I'd prefer a dark shadow to a light one. Perhaps just use the same colour as the `graph-button` shadows?

Reply:
The contrast between the white sidebar and reset selections makes the shadow look way darker than it already is so I went with the standard grey like the reset view button.
<img width="545" alt="Screen Shot 2021-07-27 at 10 14 16 AM" src="https://user-images.githubusercontent.com/32244859/127169534-bc062a23-ea91-4941-b602-0da2ab853dcd.png">